src/style.css

```diff
@@ -0,0 +1,36 @@
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

Comment:
You have styles for `.card` class here and in `card.css`
If card is a block - all related styles (block and its elements) should be in `card.css` file

Reply:
I forgot the delete that file